hardhat.config.ts

```diff
@@ -162,6 +163,23 @@ const config: HardhatUserConfig = {
 
 if (process.env.TEST) {
   config.external!.contracts = [
+    {
```

Comment:
Interesting note: apparently, the order of these affects the deployments... It's very weird